examples/reagent/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Reagent • TodoMVC</title>
```

Comment:
Please move the title right under the `<meta charset="utf-8">` for consistency with the other apps.